tools/archetypes/jms/src/main/resources/archetype-resources/pom.xml

```diff
@@ -89,17 +89,17 @@
 
       <plugin>
         <artifactId>maven-compiler-plugin</artifactId>
-        <version>3.6.1</version>
+        <version>${maven.version}</version>
```

Comment:
we need to add `maven.version` and `java.version` properties in this POM file. The properties of the parent POM will not be available at runtime in the archetpye

Reply:
I didn't know that. will revert those changes.